src/otx/core/utils/tile_merge.py

```diff
@@ -31,20 +33,30 @@ class TileMerge(Generic[T_OTXDataEntity, T_OTXBatchPredEntity]):
     def __init__(
         self,
         img_infos: list[ImageInfo],
+        tile_size: int,
+        num_classes: int,
         iou_threshold: float = 0.45,
         max_num_instances: int = 500,
```

Comment:
```suggestion
        num_classes: int,
        tile_config: TileConfig,
```
I'd rather take `tile_config` itself. If many arguments comes from it.

Reply:
Thank you, updated